subsys/nrf_security/src/drivers/cracen/cracenpsa/src/ikg_signature.c

```
@@ -158,28 +161,40 @@ int cracen_ikg_sign_digest(int identity_key_index, const struct sxhashalg *hasha
 
 int cracen_ikg_create_pub_key(int identity_key_index, uint8_t *pub_key)
 {
-	int status;
+	int status = SX_ERR_RETRY;
 	struct sx_pk_acq_req pkreq;
+	int i = 0;
+
+	/* The use of MAX_ECDSA_ATTEMPTS here is as an arbitrary value.
+	 * Needed to have a max value for this while loop and that was already defined.
+	 */
```

kinda confusing, I'd have used a separate define specially for this

---

Why, the MAX_ECDSA_ATTEMPTS is always just a arbitrary value to make sure we don't get caught in a infinite loop of HW failure. Just specifying it here since normally we don't have any errors.

---

Just confusing to use a macro whose name doesn't fit how you use it, cause here it's not about ECDSA at all right?